ipatests/test_integration/test_installation.py

```diff
@@ -1075,3 +1075,60 @@ def test_replica_install_against_server(self):
                                             self.replicas[0].hostname],
                                            stdin_text=dirman_password)
         assert self.replicas[0].hostname not in cmd.stdout_text
+
+
+class TestIpaJoin(IntegrationTest):
+
+    num_clients = 2
```

Comment:
Can we not use single client here? 
First un-enrolling the client then enrolling the same with bad password and correct password?